source/about/cloud-dedicated.rst

```diff
@@ -0,0 +1,120 @@
+Cloud Dedicated
```

Comment:
would be a good idea to add a reference architecture here? What do you think?

Reply:
@spirosoik Yes! Does it make sense to add the Confluence architecture diagram here?